scripts/src/main/resources/scripts/command/jmc

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+JMC_HOME="${DEVON_IDE_HOME}/software/jmc"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
+
+# Call jmc with specified arguments.
+function doRun() {
+  if [ -z "$(ls -A "${JMC_HOME}")" ]
+  then
+    doSetup silent
+  fi
+  doRunCommand "${JMC_HOME}/jmc ${*} &"
+}
+
+# JDK Mission Control installation
+function doSetup() {
+  doInstall "jmc" "${JMC_VERSION}" "${1}" "" "${JMC_HOME}"
+  doRunCommand "mkdir -p ${JMC_HOME}"
+  doRunCommand "mv -f '${JMC_HOME}/JDK Mission Control/' ${JMC_HOME}"
```

Comment:
this will move/rename the folder `JDK Mission Control` in `${JMC_HOME}` itself.
I expected it to do nothing but as I tested it, it renamed the folder from `JDK Mission Control` to `'JDK Mission Control'`.
You instead need to move the contents of that folder.

Reply:
Done.